department-of-veterans-affairs/vets-website
# Pension 76771 update dependent address

## Summary
Update 'Child Address' page on the Pension Benefits form (21P-527EZ)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Go to the 'Dependents' page in the 'Household information' chapter
3. Add a dependent that doesn't live with you
4. Verify the 'Child address' page is displayed
5. Verify that an address, full name and support are required
6. Verify that the data is saved to the api and redux
7. Continue to the 'Review application' page
8. Verify that the data is viewable and editable
9. Verify that the appplication is able to be submitted

## Acceptance criteria
[Issue in Github - 76772](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76772)

## Related issue(s)
n/a

## Testing done

## Screenshots
| Before | After |
| ------ | ------ |
|![localhost_3001_pension_application_527EZ_form-saved (15)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/2c8f1136-6a4a-4468-8b42-72d9450fa830)|![localhost_3001_pension_application_527EZ_form-saved (14)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/559287e6-8834-4bff-b62f-030721506d7e)



## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling


### Authentication
